### DISCUSSION TITLE
[v5] Heroku + AdonisV5

### DISCUSSION THREAD

**Discussion by brenofreire** [adonisjs/core#1962] [Help] [answered]
I never deployed a Adonis application, and I need a proof of concept. I was thinking about using Heroku as service.

I tried to deploy it but I didn't get any success.

What I tried: 
1. `npm run build --production`
2. I copied the build project to a heroku repository
3. push it to heroku

The application showed a "log" that it was running but the url is returning error like it was wrongly installed.

**Comment by thetutlage** [comment, marked as answer]
The information is not enough to provide any help. What error do you see?

**Reply by brenofreire** [reply]
Apparently my installation of adonis was not generating the correct version of the "production build". 

I created another project using the Adonis CLI and copied the controllers, models, migrations. Then I built it again, commited using the heroku cli and it worked. 

I am planning to do a tutorial on how to deploy a Adonis V5 project on heroku from the very beginning. Thanks!

**Reply by mateustech** [reply]
I have the same problem, if I can talk about how I would solve it.

I would be very grateful.

**Reply by brenofreire** [reply]
Hey @mateustech, how are you? I hope you're fine. 

In my case I LITERALLY re-created the project because for some reason it was generating a wrong build version.

So what you could do is creating another Adonis project, copy the main files (controllers, models, etc) and see what happens.
